src/style.css

```diff
@@ -0,0 +1,109 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  list-style: none;
+}
+
+html {
+  background-color: #f5f1f1;
+}
+
+.section_todo {
+  margin: 50px 10% 0 10%;
+  background-color: #fff;
+  box-shadow: 5px 5px 5px #d8dad8;
+}
+
+#todo-list {
+  padding: 16px 12px;
```

Comment:
@HFG43 Hernan! 😃 , I have noticed you used `rem` for the `font-size` properties, so since you do, you can also use `rem` for both `margin` and `padding`s! 👍🏼

Reply:
@ITurres great advice!! Had changed all paddings and margins to rem units!